receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -300,6 +313,16 @@ metrics:
       value_type: int
     attributes:
       - k8s.container.status.state
+  k8s.container.status.reason:
+    enabled: false
+    description: Experimental metric, may experience breaking changes. Describes the number of K8s containers that are currently in a state for a given reason. All possible container state reasons will be reported at each time interval to avoid missing metrics. Only the value corresponding to the current state reason will be non-zero.
+    unit: "{container}"
+    sum:
+      aggregation_temporality: cumulative
+      monotonic: false
+      value_type: int
+    attributes:
+      - k8s.container.status.reason
```

Comment:
I guess that yields the same question raised by @TylerHelmuth at https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/40926#discussion_r2200989080 and should be addressed at https://github.com/open-telemetry/semantic-conventions/pull/2472. Should we wait for an outcome from this first before moving on with this :) ?